spec/controllers/groups_controller_spec.rb

```diff
@@ -109,6 +114,11 @@
           delete :remove_group, params: { grouping_id: [grouping.id], assignment_id: assignment }
         end
 
+        it 'should return the :unprocessable_entity status code' do
+          # expect(response).to have_https_status(:unprocessable_entity)
```

Comment:
@devan-srinivasan compare this line *very carefully* with Line 93 (which you added in an above test). It might help you to copy and paste so that the lines are side by side.

Reply:
oh my god, haha such a simple mistake. digging through source code was unnecessary!